frontend/utilities/helpers.tsx

```diff
@@ -1,10 +1,22 @@
-import { isEmpty, flatMap, omit, pick, size, memoize, reduce } from "lodash";
+import React from "react";
```

Comment:
The extension for this should stay `.ts` because it's not a proper component, even though some functions return TSX.